deltachat-ios/Controller/ChatListController.swift

```diff
@@ -674,6 +674,7 @@ class ChatListController: UITableViewController {
         }
         titleView.isUserInteractionEnabled = !tableView.isEditing
         titleView.sizeToFit()
+        updateMarkArchivedReadButton()
```

Comment:
that should go to the `if isArchive` branch above - otherwise, the button is created and (expensively) updated without reasons in the normal chatlist.